src/models/events/CreateEvent.ts

```diff
@@ -14,7 +14,7 @@ export interface PreviousRoomInfo {
     /**
      * The last known event ID in the old room.
      */
-    event_id: string;
+    event_id?: string;
```

Comment:
Can you state in which conditions it's optional in the docs? I.e. room versions >= 12

Reply:
Good point. Added a brief comment to the docstring with 01cb5b7